Source/Foundation/bsfCore/Image/BsPixelUtil.cpp

```diff
@@ -2034,12 +2034,12 @@ namespace bs
 
 		// Ensure unused components are at the end, after sort
 		if (pfd.componentCount < 4)
-			compData[4].shift = 0xFF;
+			compData[3].shift = 0xFF;
```

Comment:
Write paste end error?

Reply:
well, both of those looked like off-by-one errors to me :)